### ISSUE TITLE
a11y: 'Schemes' label is not associated with its respective control

### ISSUE THREAD

**Issue by nwpappas** [swagger-api/swagger-ui#8880]
### Q&A (please complete the following information)
 - OS: all
 - Browser: all

### Describe the bug you're encountering
Label 'Schemes' is not associated with its respective control i.e. 'http' dropdown button.

### To reproduce...

1. Open URL: https://petstore.swagger.io/
1. Navigate on 'http' dropdown
1. Press F12 to open the developer tool
1. Verify that Label 'Schemes' should be associated with their respective form field or not.

### Expected behavior
Label 'Schemes' should be associated with its respective control i.e. 'http' dropdown button. Screen Reader should announce it: 'Schemes Combobox http collapsed'.

### Screenshots
![image](https://github.com/swagger-api/swagger-ui/assets/2529208/9de10c49-a561-45bd-9064-921616ec06b5)

### Additional context or thoughts
In the absence of label information for form elements, then users' dependence with screen reader is not able to understand the purpose of UI component and could give-up to use the functionality.

**Comment by morsko1** [comment]
@char0n, looks like fixed by https://github.com/swagger-api/swagger-ui/pull/9563
